e2etest/bgp_test.go

```diff
@@ -74,7 +96,11 @@ var _ = ginkgo.Describe("BGP", func() {
 
 		hostport := net.JoinHostPort(ingressIP, port)
 		address := fmt.Sprintf("http://%s/", hostport)
-		_, err = runCommand(true, "wget", "-O-", "-q", address, "-T", "60")
+		useDocker := true
+		if skipDockerCmd {
+			useDocker = false
+		}
+		_, err = runCommand(useDocker, "wget", "-O-", "-q", address, "-T", "60")
```

Comment:
Might be good to add some kind of log message so we can see what port is being used. (and maybe if docker is being used)

Reply:
Add message to mention to use docker, but not added port because Service still uses port 80 because I only added `targetPort` to specify pod's port.

Please see Kubernetes document for the services. https://kubernetes.io/docs/concepts/services-networking/service/#defining-a-service

Reply:
ok, I didn't get that on the first read actually. I see now.